setup.py

```diff
@@ -35,7 +35,7 @@ def load_requirements(*requirements_paths):
     requirements = set()
     for path in requirements_paths:
         requirements.update(
-            line.split('#')[0].strip() for line in open(path).readlines()
+            line.split('#')[0].strip() for line in open(path, encoding="utf8").readlines()
```

Comment:
pylint version 2.10.0 deprecated the use of open() without explicitly specifying any encoding. See PyCQA/pylint#3826 for details.